specification/network/resource-manager/readme.md

```diff
@@ -28,38 +28,7 @@ These are the global settings for the Network API.
 title: NetworkManagementClient
 description: Network Client
 openapi-type: arm
-tag: package-2018-05
-```
-
-### Tag: package-2018-05

```

Comment:
You can keep this tag, just default the tag back to `package-2018-04`

Reply:
Let's publish it in May to avoid misunderstanding? :)

Reply:
There is already a directory for May created in master. Wouldn't help just removing references here. Let's keep this as is and simply default the tag to April

Reply:
done